database/3ds/get_friend_persistent_infos.go

```diff
@@ -44,6 +40,7 @@ func GetFriendPersistentInfos(user1_pid uint32, pids []uint32) []*friends_3ds_ty
 				friendedAtTime = uint64(time.Now().Unix())
 			} else {
 				globals.Logger.Critical(err.Error())
+				continue
```

Comment:
This is another one where I'm not sure if we should continue to process data if an error occurs. Tagging @hauntii here as well